tendrl/ceph_integration/manager/user_request.py

```diff
@@ -237,6 +237,43 @@ def _submit(self, commands=None):
         return ceph.rados_commands(self.fsid, self._cluster_name, commands)
 
 
+class ECProfileRequest(UserRequest):
```

Comment:
My point here is that anything which uses ceph.rados_commands you have RadosRequest which is inherited from UserRequest. Similarly you can create a common class for anything which use ceph.ceph_command. Otherwise we will end up creating multiple classes for the same thing